packages/api-clients/open-api/bffApi.yml

```
@@ -15616,6 +15615,29 @@ components:
       required:
         - results
         - pagination
+    PurposeDelegation:
```

This looks like a generic delegation model with no references to any purpose, maybe we could rename it into `Delegation` or `CompactDelegation`?
```suggestion
    CompactDelegation:
```

---

There's already both a `Delegation` and a `CompactDelegation` in the BFF API, both having a lot more properties than what we need here.
I guess we could call it something like `DelegationWithCompactTenants`

---

Makes sense!